File: Backend/Backend/Mod.Products.Domain/Entities/Supplier.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Mod.Products.Domain.Entities
{
    public class Supplier
    {
        public int Id { get; set; }
        public string SupplierCode { get; set; } = string.Empty;
        public string CompanyName { get; set; } = string.Empty;
        public string TaxIdentification { get; set; } = string.Empty;
        public string? Email { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Address { get; set; }
        public bool IsActive { get; set; } = true;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: frontend/src/app/core/models/supplier.ts
export interface GetSupplierDto {
  id: number;
  supplierCode: string;
  companyName: string;
  taxIdentification: string;
  email?: string;
  phoneNumber?: string;
  address?: string;
  isActive: boolean;
}

export interface CreateSupplierDto {
  supplierCode: string;
  companyName: string;
  taxIdentification: string;
  email?: string;
  phoneNumber?: string;
  address?: string;
}